README.md

```diff
@@ -79,16 +79,13 @@ the working group does not automatically result in membership in these
 teams. These teams are:
 
 * Releasers team
-* LTS team
 * CITGM team
 
 The `releasers` team is entrusted with the secrets and CI access to be able
 build and sign releases. **Additions to the releasers team must be approved
-by the TSC following the process outlined in GOVERNANCE.md.**
-
-The Long Term Support (LTS) team manages the process/content of LTS releases
-and the required backporting for these releases. Additions to the LTS
-team needs sign off from the rest of the LTS team.
```

Comment:
So this is an interesting reality vs ideal situation. For as long as I have been involved with the Release WG the LTS team has not been the team with permissions to do releases (even LTS releases) -- that's the releasers team. It is supposed to be the team that discusses/gets to decide LTS policy/process/content. It is not a requirement that all members of the Release WG become releasers.

It's a reasonable question as to whether we still need a LTS team distinct from the whole Release WG. I know I've pinged the LTS team in the past. 

Release, like a number of other Working Groups, has the challenge of dwindling participation over time punctured by occasional drives to increase membership.

Reply:
I agree. We discussed a not long time ago that the distinction between backporters/lts-team might not be necessary anymore since all the releasers fall in the same group.

Reply:
Some related conversations in https://github.com/nodejs/Release/issues/547.